vllm/worker/habana_model_runner.py

```diff
@@ -1863,17 +1863,21 @@ def execute_model(
             logits_ids_list = []
             logits_tensor = None
             logits_tensor_list = []
+            prev_logits_device = "hpu"
```

Comment:
this will break when running on fake_hpu. Perhaps we can take the device from the model or from input_tokens?